extension/src/paymentHandler/payment-utils.js

```diff
@@ -59,6 +59,15 @@ function createChangeTransactionStateAction(transactionId, transactionState) {
   }
 }
 
+function createChangeTransactionTimestampAction(transactionId) {
+  const currentTimestamp = new Date()
```

Comment:
When we make a request to Adyen, we have in response header the field `date`. Currently we're not using it anyhow. Why did you decide to use timestamp created in code instead?

Reply:
Not sure if it is also applicable for serverless function. Need to see any other `date` attributes provided in `paymentObject`

Reply:
As discuss, we don't need to set timestamp in extension module explicitly. We just need the event date returned from notification module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2